samples/matter/light_switch/README.rst

```diff
@@ -140,34 +141,41 @@ User interface
     :end-before: matter_door_lock_sample_led1_end
 
 LED 2:
-   Shows the state of the Bluetooth LE Direct Firmware Update (DFU) process.
-
-   * Off - Bluetooth LE is not advertising and DFU is not ready.
-   * Rapid Even Flashing (30 ms off / 170 ms on) - Bluetooth LE is advertising and the DFU process can be started.
-
-LED 3:
    Identifies the device after sending the ``identify`` command to the endpoint ``0`` from the Matter controller device.
    The LED starts blinking evenly (500 ms on/500 ms off) when the Identify command of the Identify cluster is received.
    The command's argument can be used to specify the duration of the effect.
 
-LED 1-4:
+LED 1-4 for nRF52840dk, nRF5340dk and nRF21540dk, or LED 1-2 for nRF7002dk:
    Blink in unison when the factory reset procedure is initiated.
 
 .. include:: ../lock/README.rst
    :start-after: matter_door_lock_sample_button1_start
    :end-before: matter_door_lock_sample_button1_end
 
 Button 2:
-   Controls the light on the bound lighting device.
-   Depending on how long you press the button:
-
-   * If pressed for less than 0.5 seconds, it changes the light state to the opposite one on the bound lighting device (:ref:`light bulb <matter_light_bulb_sample>`).
-   * If pressed for more than 0.5 seconds, it changes the brightness of the light on the bound lighting bulb device (:ref:`light bulb <matter_light_bulb_sample>`).
-     The brightness is changing from 0% to 100% with 1% increments every 300 milliseconds as long as **Button 2** is pressed.
+    For nRF52840dk, nRF5340dk and nRF21540dk:
+       Controls the light on the bound lighting device.
+       Depending on how long you press the button:
+       * If pressed for less than 0.5 seconds, it changes the light state to the opposite one on the bound lighting device (:ref:`light bulb <matter_light_bulb_sample>`).
+       * If pressed for more than 0.5 seconds, it changes the brightness of the light on the bound lighting bulb device (:ref:`light bulb <matter_light_bulb_sample>`).
+         The brightness is changing from 0% to 100% with 1% increments every 300 milliseconds as long as **Button 2** is pressed.
+
+    For nRF7002dk:
+    * If the device is not commissioned to a Matter network, it starts the NFC tag emulation, enables Bluetooth LE advertising for the predefined period of time (15 minutes by default), and makes the device discoverable over Bluetooth LE.
+      This button is used during the :ref:`commissioning procedure <matter_light_switch_sample_remote_control_commissioning>`.
+    * If the device is commissioned to a Matter network, it controls the light on the bound lighting device.
```

Comment:
Maybe it's worth noting what happens if this button is pressed during commissioning to be precise? I.e. it has no effect I guess.

Reply:
Isn't being in the middle of commissioning equal to not commissioned state, as it is not over yet? It will not have effect, so BLE advertising will not be enabled, as we are already connected over BLE to some other device. The same situation would be pressing button in the middle of any other connection e.g DFU over BLE, so I'm not sure. I feel such complex cases may lead that user will be a little bit confused.